tree/ntuple/v7/test/rfield_blob.cxx

```diff
@@ -0,0 +1,33 @@
+#include "ntuple_test.hxx"
+
+#include <TMemFile.h>
+
+TEST(RField, Blob)
+{
+   FileRaii fileGuard("test_ntuple_rfield_blob.root");
+
+   {
+      auto model = RNTupleModel::Create();
+      auto ptrBlob = model->MakeField<ROOT::Experimental::RNTupleBLOB>("blob");
+      auto writer = RNTupleWriter::Recreate(std::move(model), "f", fileGuard.GetPath());
+
+      TMemFile f("buffer", "CREATE");
+      std::string str = "x";
+      f.WriteObject(&str, "string");
+      f.Close();
+      auto data = std::shared_ptr<unsigned char[]>(new unsigned char[f.GetSize()]);
+      f.CopyTo(data.get(), f.GetSize());
+      ptrBlob->Set(data, f.GetSize());
```

Comment:
It feels a bit odd to pass the container and the size separately ...

Reply:
Not sure I understand. Can you elaborate?

Reply:
An alternative would be
```
auto data = std::shared_ptr<std::vector<std::byte>>(new std::vector<std::byte>{f.GetSize()});
f.CopyTo(data);
ptrBlob->Set(data);
```
or
```
std::vector<std::byte> data { f.GetSize() };
f.Copy(data);
ptrBlob->Set(std::move(data));
```